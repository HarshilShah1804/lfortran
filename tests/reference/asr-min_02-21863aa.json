{
    "basename": "asr-min_02-21863aa",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/min_02.f90",
    "infile_hash": "b683622b846877f83e88b48c637b382d1a967a1d5bf15622d056ab8e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-min_02-21863aa.stdout",
    "stdout_hash": "dac1040b6140ff2921906b8fab9931fad237f2b3fa94cea3b656ae8c",
    "stderr": "asr-min_02-21863aa.stderr",
    "stderr_hash": "af77179f6e6513e1312ab2c0128d0e5c62b130b7e05dd68748884d6b",
    "returncode": 0
}