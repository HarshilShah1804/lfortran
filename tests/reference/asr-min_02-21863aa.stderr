warning: Different kinds of args in max0 is a non-standard extension
 --> tests/../integration_tests/min_02.f90:4:14
  |
4 |     print *, min(y, z)
  |              ^^^^^^^^^ help: ensure all arguments have the same kind to make it standard

warning: Different kinds of args in max0 is a non-standard extension
 --> tests/../integration_tests/min_02.f90:5:40
  |
5 |     print *, "kind of min(y,z):", kind(min(y,z))
  |                                        ^^^^^^^^ help: ensure all arguments have the same kind to make it standard

warning: Different kinds of args in max0 is a non-standard extension
 --> tests/../integration_tests/min_02.f90:6:14
  |
6 |     if (kind(min(y,z)) /= 8) error stop "Incorrect kind for min"
  |              ^^^^^^^^ help: ensure all arguments have the same kind to make it standard
